dyo/dyo.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	cli "github.com/urfave/cli/v2"
+)
+
+func main() {
```

Comment:
If I run with a random command, it will exit with a random(? for me) status. It would be nice if an unknown command given by the user, show the help page. Same with the flags.

Reply:
Will address this soon!

Reply:
fixed